docs/coordinates/index.rst

```diff
@@ -367,9 +376,9 @@ high-level |skycoord| method - see :ref:`astropy-coordinates-rv-corrs`)::
 
     >>> from astropy.time import Time
     >>> obstime = Time('2017-2-14')
-    >>> target = SkyCoord.from_name('M31')  # doctest: +REMOTE_DATA
-    >>> keck = EarthLocation.of_site('Keck')  # doctest: +REMOTE_DATA
-    >>> target.radial_velocity_correction(obstime=obstime, location=keck).to('km/s')  # doctest: +REMOTE_DATA +FLOAT_CMP
+    >>> target = SkyCoord.from_name('M31')  # doctest: +SKIP
+    >>> keck = EarthLocation.of_site('Keck')  # doctest: +SKIP
+    >>> target.radial_velocity_correction(obstime=obstime, location=keck).to('km/s')  # doctest: +FLOAT_CMP  +REMOTE_DATA
```

Comment:
If we can figure out the IERS stuff, we can remove `+REMOTE_DATA` here too but that is a battle for another day.